cmd/collector/app/span_processor_test.go

```diff
@@ -807,3 +816,104 @@ func TestSpanProcessorWithOnDroppedSpanOption(t *testing.T) {
 	require.EqualError(t, err, processor.ErrBusy.Error())
 	assert.Equal(t, []string{"op3"}, droppedOperations)
 }
+
+func optionsWithPorts(portHttp string, portGrpc string) *cFlags.CollectorOptions {
+	opts := &cFlags.CollectorOptions{
+		OTLP: struct {
+			Enabled bool
+			GRPC    configgrpc.ServerConfig
+			HTTP    confighttp.ServerConfig
+		}{
+			Enabled: true,
+			HTTP: confighttp.ServerConfig{
+				Endpoint: portHttp,
+			},
+			GRPC: configgrpc.ServerConfig{
+				NetAddr: confignet.AddrConfig{
+					Endpoint:  portGrpc,
+					Transport: confignet.TransportTypeTCP,
+				},
+			},
+		},
+	}
+	return opts
+}
+
+func TestOTLPReceiverWithV2Storage(t *testing.T) {
```

Comment:
per the issue description, please have this test defined in otlp_receiver_test

https://github.com/jaegertracing/jaeger/blob/441c2747f55adb8d0c3ba06058f3f56400eacb4e/cmd/collector/app/handler/otlp_receiver_test.go#L60

Reply:
if you cannot import, then ok to leave as is